src/movepick.cpp

```diff
@@ -225,7 +222,7 @@ Move MovePicker::next_move(bool skipQuiets) {
       /* fallthrough */
 
   case BAD_CAPTURE:
-      return select<Next>(Any);
+      return select<Next>([&](){ return true; });
```

Comment:
Can't these just be 
```C++
return select<Next>([](){ return true; });
```
w/o the & ?

Reply:
Works on my machine, but not for the other filters.